src/index.html

```diff
@@ -1,12 +1,317 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__container">
+
+        <div class="header__logo">
+          <a href="#home">
+            <img src="./images/logo.png" alt="Moyo">
+          </a>
+        </div>
+
+        <nav class="header__menu">
+          <ul class="header__list">
+            <li class="header__item">
+              <a href="#apple" class="header__link is-active">Apple</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#samsung" class="header__link">Samsung</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#smartphones" class="header__link">Smartphones</a>
+            </li>
+
+            <li class="header__item">
+              <a
+                data-qa="nav-hover"
+                href="#laptops&computers"
+                class="header__link"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="header__item">
+              <a href="#gadgets" class="header__link">Gadgets</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#tablets" class="header__link">Tablets</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#photo" class="header__link">Photo</a>
+            </li>
+
+            <li class="header__item">
+              <a href="#video" class="header__link">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__container">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__subtitle">Product code: 195434</p>
+
+          <div class="card__rating">
+            <ul class="stars stars--4">
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+              <li class="stars__star"></li>
+            </ul>
+
+            <p class="card__reviews">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__cost">$2,199</p>
+          </div>
+
+          <a
+              href="#basket"
+              class="card__button"
+              data-qa="card-hover">buy
+          </a>
```

Comment:
```suggestion
          <a
            href="#basket"
            class="card__button"
            data-qa="card-hover">buy
          </a>
```
code style - indentation